src/main/java/io/trino/tpcds/column/CustomerColumn.java

```diff
@@ -41,7 +41,7 @@
     C_BIRTH_COUNTRY(varchar(20)),
     C_LOGIN(character(13)),
     C_EMAIL_ADDRESS(character(50)),
-    C_LAST_REVIEW_DATE_SK(IDENTIFIER);
+    C_LAST_REVIEW_DATE(IDENTIFIER);
```

Comment:
since this is identifier (reference to other table), it most likely should have `_sk` suffix

Reply:
Yea, the changelog reflects it.

> 09-26-2017 v2.6.0
> Replaced two occurrences of "c_last_review_date" with "c_last_review_date_sk" to be consistent 
with Table 2-14 (Customer Table Column Definitions) in section 2.4.7 of the specification (fogbugz 
2046).

Reply:
yes, according to https://www.tpc.org/tpc_documents_current_versions/pdf/tpc-ds_v2.1.0.pdf
this should be: `c_last_review_date_sk identifier`